### DISCUSSION TITLE
October|Technical Manager Monthly Report

### DISCUSSION THREAD

**Discussion by casinoyoda** [ringdao-community-guild/discussion#51] [General]
Last month, my primary responsibilities revolved around the daily upkeep of DCDao's infrastructure and providing technical support. Specifically, I focused on:

- Processing applications and [issuing Profiles NFTs](https://darwinia.subscan.io/tx/0x5becf1284a64b5d88f83afa65f17653236156d6d230b2c2cead8daa5101857b4). 
- Organizing the DSafe code repository.
- Investigate the Darwinia GPT
- Daily maintaining the message bridge between Discord and Slack.

**Comment by hackfisher** [comment]
@casinoyoda please take a look at:

https://github.com/orgs/dcdao/discussions/55
